packages/node_modules/@ciscospark/spark-core/src/spark-core.js

```diff
@@ -49,12 +51,14 @@ const interceptors = {
 const preInterceptors = [
   `ResponseLoggerInterceptor`,
   `RequestTimingInterceptor`,
+  `RequestEventInterceptor`,
```

Comment:
By adding the same interceptor to pre and post, you're going to end up emitting events twice. You want a RequestEvent interceptor in pre and a ResponseEvent interceptor in post.